.travis.yml

```diff
@@ -16,7 +16,9 @@ jobs:
     - name: Linux | arm64 | Python 3.6
       python: 3.6
       services: docker
-      arch: arm64
+      arch: arm64-graviton2
+      group: edge
+      virt: lxd
```

Comment:
Any reason to choose `lxd` rather than `vm` here ?
I think that `vm`, while they are a bit slower to start are more stable and are a bit more appropriate for docker usage. They're are all kinds of things going bad on ppc64 lxd, one of the reason is docker is running on top of lxd does not always agree with the lxd configuration.